std/assembly/symbol.ts

```diff
@@ -7,10 +7,22 @@ var nextId: usize = 12; // Symbol.unscopables + 1
 @unmanaged export class symbol {
   toString(): string {
     var id = changetype<usize>(this);
-    if (idToString !== null && idToString.has(id)) {
-      return "Symbol(" + idToString.get(id) + ")";
+    var str: string | null = null;
+    switch (id) {
+      case 1:  { str = "hasInstance"; break; }
+      case 2:  { str = "isConcatSpreadable"; break; }
+      case 3:  { str = "isRegExp"; break; }
+      case 4:  { str = "match"; break; }
+      case 5:  { str = "replace"; break; }
+      case 6:  { str = "search"; break; }
+      case 7:  { str = "species"; break; }
+      case 8:  { str = "split"; break; }
+      case 9:  { str = "toPrimitive"; break; }
+      case 10: { str = "toStringTag"; break; }
+      case 11: { str = "unscopables"; break; }
     }
-    return "Symbol()";
+    if (idToString !== null && idToString.has(id)) str = idToString.get(id);
```

Comment:
Looks like this should be the default case of the switch above

Reply:
Yeah, make sense